flask_api/tests/test_mediatypes.py

```diff
@@ -6,7 +6,7 @@
 
 class MediaTypeParsingTests(unittest.TestCase):
     def test_media_type_with_params(self):
-        media = MediaType('application/xml; schema=foobar, q=0.5')
+        media = MediaType('application/xml; schema=foobar; q=0.5')
```

Comment:
I found that [RFC 2616 Section 14](http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html) seemed to indicate that parameters are separated with commas, but the grammar is not very clear.
I also referred to [some other W3C documentation](http://www.w3.org/Protocols/HTTP/HTRQ_Headers.html) for confirmation.